tests/unit/array/mixins/test_getset.py

```diff
@@ -493,9 +515,13 @@ def test_getset_subindex(storage, config):
         ('qdrant', {'n_dim': 3, 'distance': 'euclidean'}),
         ('elasticsearch', {'n_dim': 3, 'distance': 'l2_norm'}),
         ('sqlite', dict()),
+        ('redis', {'n_dim': 3, 'distance': 'L2', 'flush': True}),
     ],
 )
 def test_init_subindex(storage, config):
+    if storage == 'redis':
+        ensure_gc
```

Comment:
I don't get how this works
why don't we make it a function instead of fixture and just call it ?